cmd/main.go

```diff
@@ -76,32 +53,13 @@ func main() {
 
 	wg := &sync.WaitGroup{} // Goroutines can add themselves to this to be waited on
 
-	controllerConfig := ControllerConfig()
-
-	if configMapName != (spec.NamespacedName{}) {
-		configMap, err := controllerConfig.KubeClient.ConfigMaps(configMapName.Namespace).Get(configMapName.Name)
-		if err != nil {
-			panic(err)
-		}
-
-		configMapData = configMap.Data
-	} else {
-		log.Printf("No ConfigMap specified. Loading default values")
-	}
-	if configMapData["namespace"] == "" { // Namespace in ConfigMap has priority over env var
-		configMapData["namespace"] = podNamespace
-	}
-	if noDatabaseAccess {
-		configMapData["enable_database_access"] = "false"
-	}
-	if noTeamsAPI {
-		configMapData["enable_teams_api"] = "false"
+	config.RestConfig, err = k8sutil.RestConfig(KubeConfigFile, OutOfCluster)
```

Comment:
it looks like the RestConfig is the only reason why k8util is imported from main. We can move it completely to the controller. Moreover, I think we don't need to store it anywhere, since the sole reason of its existence so far is to run exec.